exporter/awsprometheusremotewriteexporter/auth_test.go

```diff
@@ -109,7 +110,7 @@ func TestLeakingBody(t *testing.T) {
 			return newSigningRoundTripperWithCredentials(authConfig, awsCreds, next, sdkInformation)
 		},
 	}
-	client, _ := setting.ToClient()
+	client, _ := setting.ToClient(componenttest.NewNopHost().GetExtensions())
```

Comment:
If you prefer, you can use something like `map[config.ComponentID]component.Extension{}` instead.

Reply:
That's also a great option, @jpkrohling! Which way do you believe is the correct convention for us to use moving forward? I know we typically use `componentest` when possible, so I thought following that pattern made the most sense at the time of this PR.